.gitignore

```diff
@@ -1,3 +1,32 @@
+# Node artifact files
```

Comment:
Remove this, this is not a Javascript application, and `dist/` is duplicated below

Reply:
Done